pkg/virtualization/core/service/virtualmachine.go

```diff
@@ -272,3 +272,31 @@ func (vmms *VirtualSystemManagementService) SetMemoryMB(vm *virtualsystem.Virtua
 	err = vmms.ModifyVirtualSystemResourceEx(mem.WmiInstance, -1)
 	return
 }
+
+// Re-implementation of Disable-VMConsoleSupport Cmdlet
+// Removes the Synthetic Mouse, Synthetic Keyboard, and Synthetic Display devices from a VM
+func (vmms *VirtualSystemManagementService) RemoveHIDDevices(vm *virtualsystem.VirtualMachine) (err error) {
```

Comment:
can you add a unit test to _test.go

Reply:
could you please point me to the documentation for the tests? I tried running them through `go test` and they fail

Reply:
nvm, added